src/reducer/index.js

```diff
@@ -0,0 +1,41 @@
+import { ADD_REMINDER, DELETE_REMINDER, CLEAR_REMINDER } from '../constants/constants';
+import { bake_cookie, read_cookie } from 'sfcookies';
+
+
+const reminder = (action) => {
+    let { text, dueDate} = action;
+    return {
+        id: Math.random(),
```

Comment:
You might want to be more descriptive with this function's name to clearly convey what it does. Perhaps you could call it something like `createReminder` instead.